src/CommandingOfficers/AW4/RuinedCommander.java

```diff
@@ -208,14 +209,16 @@ public boolean isInZone(UnitContext uc)
 
     for( Unit cou : COUs )
     {
+      if( !army.myView.isLocationValid(cou.x, cou.x) )
+        continue; // Loaded/dead COUs shouldn't boost the top left, derp
```

Comment:
Hah. Maybe our "not present" coordinates should have been farther away.

Reply:
Arguably, but it's also arguably a logic error to not account for "off map" at all, so...